cosmovisor/README.md

```diff
@@ -20,6 +20,7 @@ All arguments passed to the `cosmovisor` program will be passed to the current d
 * `DAEMON_NAME` is the name of the binary itself (e.g. `gaiad`, `regend`, `simd`, etc.).
 * `DAEMON_ALLOW_DOWNLOAD_BINARIES` (*optional*), if set to `true`, will enable auto-downloading of new binaries (for security reasons, this is intended for full nodes rather than validators). By default, `cosmovisor` will not auto-download new binaries.
 * `DAEMON_RESTART_AFTER_UPGRADE` (*optional*), if set to `true`, will restart the subprocess with the same command line arguments and flags (but with the new binary) after a successful upgrade. By default, `cosmovisor` stops running after an upgrade and requires the system administrator to manually restart it. Note that `cosmovisor` will not auto-restart the subprocess if there was an error.
+* `UNSAFE_UPGRADE` (defaults to `false`), if set to `true`, will backup the data before trying the upgrade. This is useful in case of failures and when needed to rollback. It is advised to use backup option, i.e., `UNSFAE_UPGRADE=false`
```

Comment:
All other env vars start with `DAEMON_`, should this one too?

Reply:
Yes IMO

Reply:
Hmm, how about `DAEMON_UNSAFE_SKIP_BACKUP`?

Reply:
Let's prefix all env vars with COSMOVISOR (in a separate PR)

Reply:
`DAEMON_HOME` and `DAEMON_NAME` are actually refering to the original daemon ex: `gaiad` and not `cosmovisor`, so renaming them might confuse.

Reply:
Feels weird to call it UNSAFE_UPGRADE, instead of `MAKE_BACKUPS`

Reply:
@ValarDragon  - the idea is that by default we are doing backup. So if you want to do something unsafe (not having a backup) then you need to write this "UNSAFE...."

Reply:
renamed it to `UNSAFE_SKIP_BACKUP` for redability

Reply:
Thanks! Thats a much better name imo